test/scripts/compare_test_results.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+"""Script to compare output of [Core]NEURON runs.
+
+This utility is used to compare the results of different NEURON and CoreNEURON
+runs and determine whether or not they are consistent with one another.
+Positional arguments are of the form testname::type1::path1[::type2::path2]...
+
+  Typical usage example:
+
+  compare_test_results.py test1::asciispikes::path1 test2::asciispikes::path2
+"""
+import collections
+import itertools
+import subprocess
+import sys
+from pprint import pprint
```

Comment:
unused

Reply:
Removed in 6fc0e7bbbd33ccd864e93723885ad3ecd8a621e5.